rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -1194,20 +1291,18 @@ def run():
                                        default=None, required=False)
     process_runner_parser.add_argument("--csv", help="A file specifying multiple manifests to process",
                                        default=None, required=False)
-    process_runner_parser.add_argument("--dryrun", help="for testing", default=False, action="store_true")  # noqa
-
-    # Backfill GenomicFileProcessed UploadDate
-    upload_date_parser = subparser.add_parser("backfill-upload-date")
```

Comment:
Is this subparser still needed to trigger the backfill-upload-date if-block later?

Reply:
It looks like this line is still needed. I'll add back in.